tgarchive/example/template.html

```diff
@@ -14,16 +15,16 @@
 	<meta property="og:description" content="{{ meta_description }}" />
 	<meta property="og:type" content="website" />
 	<meta property="og:url" content="{{ config.site_url }}/.html" />
-	<meta property="og:image" content="{{ config.site_url }}/static/thumb.png" />
+	<meta property="og:image" content="{{ config.site_url }}/static/thumb.png" />{%
+	if config.publish_rss_feed %}
 
-	{% if config.publish_rss_feed %}
-		<link rel="alternate" type="application/rss+xml" title="RSS feed " href="index.xml" />
-		<link rel="alternate" type="application/atom+xml" title="Atom feed " href="index.atom" />
-	{% endif %}
+	<link rel="alternate" type="application/rss+xml" title="RSS feed " href="index.xml" />
+	<link rel="alternate" type="application/atom+xml" title="Atom feed " href="index.atom" />{%
+	endif %}
 
 	<link rel="preconnect" href="https://fonts.gstatic.com">
-	<link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600&display=swap" rel="stylesheet"> 
```

Comment:
can we remove links to google?

Reply:
Users can easily remove this if they don't want it once the template it generated. I think it's okay for it to be there by default.